setup.py

```diff
@@ -95,13 +107,13 @@ def build_extension(self, ext):
                 cmake_args += ["-A", "x64"]
             build_args += ["--", "/m"]
         else:
-            build_args += ["--", "-j"]
+            build_args += ["--", "-j%d" % max(MIN_CPU_CORES, get_cpu_count())]
```

Comment:
```suggestion
            build_args += ["--", "-j{}".format(max(MIN_CPU_CORES, get_cpu_count()))]
```

Reply:
For single args, I find the `%` easier to read.